### PR TITLE
User Feedback Team Meeting

### PR DESCRIPTION
**Pull request by mihaiep**
minutes/notes team meeting 06-22-2018

### CODE REVIEW

File: meetings/2018-06-22.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+* Invite Node.js tooling ecosystem folks to discuss user feedback [#37](https://github.com/nodejs/user-feedback/issues/37)
+
+  * Nothing to talk about on this front, just remove label.
+  * Happy with progress that Chris is making
+
+* Pull in threads from collab summit.
+
+  * 3-5 people mention, we would like to do sessions, encourage people to create issues but I 
+    don’t think they have shown up.
+  *  main one that Dan remembers is promises
```
Comment (mhdawson) [inline meetings/2018-06-22.md]: one extra space

File: meetings/2018-06-22.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  *  Tierney suggested that we create issue.
```
Comment (mhdawson) [inline meetings/2018-06-22.md]: one extra space

File: meetings/2018-06-22.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  challenges around that and they’d like to build understanding and get feedback around promise usage and if I put words in 
+  their mouth I would think that their end game would be to see if they could influence the project more to encourage more 
+  platform level adoption and support for Promises.
+* Dan: Speaking of that we need to do at least one session, we’re going to teach you how to fish an then please run more 
+  of those sessions so you can get more feedback we will continue to amplify and support, encouraging either companies
+  that are actively using promises or companies that are struggling or unhappy with promises, to come and engage in those 
+  sessions and by having those sessions captured and the feedback discussions, that will provide the baseline inputs that 
+  the project needs to be able to get more feedback and then from there maybe a survey would be a good tool, but I think 
+  there’s probably some steps before a survey, some consensus building before a survey to be most effective.
+* Michael: In that context make sense, we can only take on so many regular meetings for imprint we’ve got the tooling one
```
Comment (mhdawson) [inline meetings/2018-06-22.md]: delete "for imprint"

File: meetings/2018-06-22.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+* Tierney: The problem it solves is we don’t have any understanding of how node is used, we understand the places it’s used,
+  we don’t understand at what scale.
+* Michael: I think once you go beyond some high-level numbers you then get into even more trickiness in terms of like can
+  the cloud providers even collect that data? There is a lot of discussions around it, but you can’t just go and collect
+  their data, you got to respect their privacy and what they’re doing. There are a lot of challenges…
+* Tierney: Grossly underreporting right now, with 2 data sets we are at the lower end like the nine million number is the 
+  lower end of the possible range, so my estimate it’s 18 million.
+  We cannot just rely on this data set as the single source truth because even the people who are talking aboutthat single
+  source of truth, have their own data sets so we need to be able to co-relate that a bit.
+* Michael: Dockers is a good one. The downloads, the docker distros are the other major sources. Are any other ones beyond that?
```
Comment (mhdawson) [inline meetings/2018-06-22.md]: Docker

File: meetings/2018-06-22.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+* Tierney: Public sources, no.
+* Michael: Deployments are quite different.
+* Tierney: Docker is a closer representation of deployments than downloads because it’s a pull and it’s effectively 
+  one person using it, there is zero way to tell if it’s a development deployment or development usage or a CI/CD usage 
+  or a production usage.
+* Michael: If you want like instances deployed like Kubernetes would probably be very once at the very beginning.
+* Tierney: You are deploying once and then things are happening, I don’t know if that should be counted as multiple deployments 
+  because you are deploying it and it’s doing that saying.
+* Michael: You could think that a cloud provider may pull a docker image once and how is caching to prove.
```
Comment (mhdawson) [inline meetings/2018-06-22.md]: remove "to prove"

**Comment by mihaiep** [comment]
Fixed. Thanks Michael for reviewing.

**Pull request merged**